setup.py

```diff
@@ -7,7 +7,7 @@
 
 extras_require={
     'test': [
-        "pytest==3.3.2",
+        "pytest>=3.6.0",
```

Comment:
Is this a strong requirement of this pull request? I generally try to touch the minimum when I PR, but maybe that one is mandatory

Reply:
Yeah. `master` is failing due to a new release:

https://github.com/ethereum/eth-account/pull/52#issuecomment-450754364

`pytest-xdist 1.25.0 has requirement pytest>=3.6.0, but you'll have pytest 3.3.2 which is incompatible.` ([1.24.1](https://github.com/pytest-dev/pytest-xdist/blob/v1.24.1/setup.py#L3) vs. [1.25.0](https://github.com/pytest-dev/pytest-xdist/blob/v1.25.0/setup.py#L3))

Reply:
Great, I was suspecting something like this, but didn't check, sorry. Thanks for your reply :beers: